### PR TITLE
ch:31845: Fix support bundle command for airgap installations

### PR DESCRIPTION
**Pull request by jala-dx**

### CODE REVIEW

File: web/src/components/troubleshoot/GenerateSupportBundle.jsx [hunk 1]
```diff
@@ -314,6 +314,37 @@ class GenerateSupportBundle extends React.Component {
                 </div>
               }
               {showRunCommand ?
+                watch.isAirgap ?
```
Comment (sgalsaleh) [inline web/src/components/troubleshoot/GenerateSupportBundle.jsx:317]: we still calling this `watch`? :) wow

Comment (GraysonNull) [reply web/src/components/troubleshoot/GenerateSupportBundle.jsx:317]: haha yeah...still in the code here and there 🤡

**Comment by GraysonNull** [comment]
Screen Shot 2021-08-12 at 5 05 37 PM

**Review by divolgin — CHANGES_REQUESTED** [review]
This should use the spec generated by kotsadm and provide instructions for installing support-bundle binary.

**Pull request closed without merging**